src/styles/blocks/card.scss

```diff
@@ -0,0 +1,128 @@
+.catalog {
+  --column-count: 1;
+
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 46px;
+  padding: 50px 40px;
+  margin: 0;
+
+  @media (min-width: 488px) {
+    --column-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    --column-count: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --column-count: 4;
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  background-color: $main-color;
```

Comment:
The variable `$main-color` is used here, but it is not defined in this file. Ensure that it is defined in a file that is imported before this one, or define it in this file if it's missing.